net-analyzer/gvm-libs/gvm-libs-20.8.1.ebuild

```diff
@@ -0,0 +1,105 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CMAKE_MAKEFILE_GENERATOR="emake"
+inherit cmake flag-o-matic toolchain-funcs
+
+DESCRIPTION="Greenbone vulnerability management libraries, previously named openvas-libraries"
+HOMEPAGE="https://www.greenbone.net/en/"
+SRC_URI="https://github.com/greenbone/gvm-libs/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+SLOT="0"
+LICENSE="GPL-2+"
+KEYWORDS="~amd64 ~x86"
+IUSE="extras ldap test radius"
```

Comment:
`ldap` and `radius` seem to have no function in this ebuild, other than control optional RDEPEND. It's forbidden by our QA policy, 
https://projects.gentoo.org/qa/policy-guide/dependencies.html#pg0001

if they are some automagic dependencies, you need to patch them out or straight depend on those packages.

Reply:
This are optional build dependencies, which can be disabled through a configure option.
But if they are enabled and not found, they will be disabled automatically by cmake.
Is this automatic disabling behavior ok or should I patch this to fail if it's enabled and not found?

Reply:
You should be able to write a logic in `src_configure` using USE flags for those. So yeah, please keep them and make them optional in the build system.